test/test_functions

```diff
@@ -955,7 +963,11 @@ load_repo_config() {
 
 
 get_millisecond_epoch() {
-  echo $(( $(date +%s%N) / 1000000 ))
+  if running_on_osx; then
+    echo $(( $(date +%s) * 1000 ))
```

Comment:
I know this is an ugly workaround, but this function is only used to get the milliseconds spent running the test, so it's not crucial.
This is because the '%N' is not supported on mac, and there is no equivalent